Example/SmileID/CodeScanner/CodeScanner.swift

```diff
@@ -0,0 +1,128 @@
+// swiftlint:disable all
```

Comment:
Does this library allow you to add it as a Podfile or using SPM?

Reply:
See my comment in the original post

> It is available only via SPM, not as a CocoaPod, which is what we use for our sample app. Therefore the code has been manually copied over to this repo (the project is MIT Licensed).